.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Open Policy Agent
+      uses: engineerd/configurator@v0.0.3
+      with:
+        name: opa
+        repo: open-policy-agent/opa
+        version: ^0.23.0
+        urlTemplate: 'https://github.com/open-policy-agent/opa/releases/download/{{version}}/opa_linux_amd64'
+        fromGitHubReleases: true
+        token: ${{ github.token }}
```

Comment:
Generally speaking I would prefer if we can avoid using third party actions where possible (in particular when they need credentials).

Would it be possible to replace our `opa` usage with the OPA docker image or replace this with a `curl` shell step?

Reply:
Oh? If you'd like I can pin to a specific commit. It should be easy however to curl if that's preferred, it just won't pick up new patch versions automatically.

Reply:
> it just won't pick up new patch versions automatically.

Thats OK, pinning to a specific version is ideal. Later on we can hook it up with the auto updater tooling we use for some of the other OPA integrations to keep it in sync with the latest stable OPA releases.

Reply:
Done!